node/src/cli.rs

```diff
@@ -144,6 +144,10 @@ pub struct Cli {
 	/// Relaychain arguments
 	#[structopt(raw = true)]
 	pub relaychain_args: Vec<String>,
+
+	/// Enable EVM tracing module on a non-authority node.
+	#[structopt(long, conflicts_with = "collator", conflicts_with = "validator")]
+	pub debug: bool,
```

Comment:
Should this go above in the `pub struct RunCmd`? It is only relevant when running a node (not when eg purging a chain, exporting genesis state, reverting blocks, etc).

If it works here, I don't have a strong preference, just something I noticed.

Reply:
Thanks, I will change it